cmd/zc_traverser_local.go

```diff
@@ -187,6 +194,9 @@ func WalkWithSymlinks(appCtx context.Context, fullPath string, walkFunc filepath
 		parallel.Walk(appCtx, queueItem.fullPath, EnumerationParallelism, EnumerationParallelStatFiles, func(filePath string, fileInfo os.FileInfo, fileError error) error {
 			if fileError != nil {
 				WarnStdoutAndScanningLog(fmt.Sprintf("Accessing '%s' failed with error: %s", filePath, fileError))
+				if errorChannel != nil {
```

Comment:
I think we need to add this code other places where azcopy silently failed. Like L217, L222, L229,